hieroplatform/src/main/java/org/hiero/sketch/spreadsheet/QuantileSketch.java

```diff
@@ -17,14 +17,21 @@
  * QuantileSketch provides two main methods:
  * - getQuantile: It creates a QuantileList from an input Table
  * - add: It combines two QuantileLists created from disjoint dataSets to create a single new
- * QuantileList, that captures Quantile information for the union.
+ *   QuantileList, that captures Quantile information for the union.
  * It stores the following objects:
- * - resolution: the desired number of quantiles.
  * - colSortOrder: the order and orientation of the columns to define the sorted order.
+ * - resolution: the desired number of quantiles.
```

Comment:
You could consider moving these comments on the respective fields. The IDE can help display the comment when you hover over the field in the code.